modular_skyrat/modules/sec_haul/code/guns/bullets.dm

```diff
@@ -54,45 +54,45 @@
 /obj/projectile/bullet/advanced/b6mm/ihdf
 	name = "6.3mm fragmentation pellet"
 	icon_state = "bullet_i"
-	damage = 15
-	bare_wound_bonus = 50
-	embedding = list(embed_chance=60, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
+	damage = 25
+	bare_wound_bonus = 20
+	embedding = list(embed_chance=40, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
 	weak_against_armour = TRUE
 
 ///////////9x19mm/////////////////
 //FMJ | JHP | IHDF | RUBBER//
 ///////////////////////////////
 /obj/item/ammo_casing/b9mm
-	name = "9x19mm peacekeeper FMJ casing"
+	name = "9mm peacekeeper casing"
 	desc = "A 9x19mm bullet casing with a low powder load to prevent breaches."
 	icon = 'modular_skyrat/modules/sec_haul/icons/guns/ammo_cartridges.dmi'
 	icon_state = "sl-casing"
 	caliber = CALIBER_9MM
 	projectile_type = /obj/projectile/bullet/advanced/b9mm
 
 /obj/projectile/bullet/advanced/b9mm
-	name = "9x19mm FMJ bullet"
-	damage = 15
-	speed = 0.8
+	name = "9mm peacekeeper bullet"
+	damage = 25
+	speed = 1.2
 
 /obj/item/ammo_casing/b9mm/hp
-	name = "9x19mm JHP bullet casing"
-	desc = "A 9x19mm jacketed hollowpoint casing."
+	name = "9mm peacekeeper frangible bullet casing"
+	desc = "A 9x19mm frangible casing."
 	icon = 'modular_skyrat/modules/sec_haul/icons/guns/ammo_cartridges.dmi'
 	icon_state = "sh-casing"
 	caliber = CALIBER_9MM
 	projectile_type = /obj/projectile/bullet/advanced/b9mm/hp
 
 /obj/projectile/bullet/advanced/b9mm/hp
-	name = "9x19mm JHP bullet"
+	name = "9mm peacekeeper frangible bullet"
 	icon_state = "bullet_h"
-	damage = 20
-	wound_bonus = 35
-	embedding = list(embed_chance=75, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
+	damage = 25
+	wound_bonus = 30
+	embedding = list(embed_chance=60, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
 	weak_against_armour = TRUE
 
 /obj/item/ammo_casing/b9mm/ihdf
```

Comment:
```suggestion
/obj/item/ammo_casing/b9mm/stamina
```

Reply:
It's not *necessary* but if it's no longer named ihdf, it shouldn't be named ihdf in the typepath, because that's making it harder to find for someone that doesn't know that's what they used to be.